src/server/index.ts

```diff
@@ -84,6 +85,7 @@ export function Server({
   authenticateCredentials,
   generateCredentials,
   https,
+  socketAdapter,
```

Comment:
There's no need to pass this to the `Server` constructor because the `SocketIO` class below is exposed to the user. So, if they want to customize the options sent they can just do this:

```js
import { SocketIO, Server } from 'boardgame.io/server';

const server = Server({
  transport: new SocketIO({ ... })
})

Reply:
Ah, I see. I don't think we need this at all, if we can pass the SocketIO instance to the transport. I will close this one.